README.md

```diff
@@ -25,12 +25,12 @@ The [Quick Start](http://python.dronekit.io/guide/quick_start.html) guide explai
 A basic script looks like this:
 
 ```python
-from dronekit import connect, VehicleMode
+from dronekit import connect
 
 # Connect to UDP endpoint.
 vehicle = connect('127.0.0.1:14550', wait_ready=True)
 # Use returned Vehicle object to query device state - e.g. to get the mode:
-print " Mode: %s" % vehicle.mode.name
+print("Mode: %s" % vehicle.mode.name)
```

Comment:
Does this actually work without importing VehicleMode?

Reply:
Yes. I just ran this test in SITL and it works as expected.
```python
from dronekit import connect
from dronekit.test import with_sitl


@with_sitl
def test_readme(connpath):
    # Connect to UDP endpoint.
    vehicle = connect(connpath, wait_ready=True)
    # Use returned Vehicle object to query device state - e.g. to get the mode:
    print("Mode: %s" % vehicle.mode.name)
    vehicle.close()
```

Output:
```
$ nosetests -svx dronekit.test.sitl.test_readme
dronekit.test.sitl.test_readme.test ... Mode: STABILIZE
ok

----------------------------------------------------------------------
Ran 1 test in 16.342s

OK
```